backend/src/main/java/fk/prof/backend/http/ApiPathConstants.java

```diff
@@ -0,0 +1,15 @@
+package fk.prof.backend.http;
+
+public final class ApiPathConstants {
+  private ApiPathConstants() {
+  }
+
+  public static final String AGGREGATOR_POST_PROFILE = "/profile";
+
+  public static final String LEADER_POST_LOAD = "/leader/load";
+  public static final String LEADER_GET_WORK = "/leader/work";
+  public static final String LEADER_PUT_ASSOCIATION = "/leader/association";
```

Comment:
The same cache is being reused for serving /leader/load and /leader/association (backendassociationstore)
The intent of both API is different. /leader/load is hit by backend machines. This API is dumb, accepts load and just serves existing associations to backend machine. /leader/association is hit by recorders(via ELB->routed to backend->routed to leader). This API actively associates the recorder to a new backend if no association found or existing association is defunct. On subsequent calls to /leader/load, backend is notified of updated associations